examples/router/webpack.config.js

```diff
@@ -1,12 +1,13 @@
+const path = require('path');
 const externals = require('frint-config').externals;
 
 module.exports = {
   entry: {
-    root: __dirname + '/root/index.js',
+    root: path.resolve(__dirname, 'root/index.js'),
   },
   devtool: 'source-map',
   output: {
-    path: __dirname + '/build/js',
+    path: path.resolve(__dirname, 'build/js'),
```

Comment:
because of individual imports in FrintJS packages (like `rxjs/Observable`), we ran into issues before when the libraries were coming from a separate `vendors.js` file.

A new key `libraryTarget` key set to `umd` helped us sort the issue out.

for e.g., you can see this Webpack config here: https://github.com/Travix-International/frint/blob/master/packages/frint-data/webpack.config.js#L24